clientdb/batch_snapshot.go

```diff
@@ -15,25 +15,26 @@ import (
 )
 
 // batch-snapshot-bucket
-//         |
-//         |-- batch-snapshot-pending-key: <batch snapshot>
-//         |
-//         |-- batch-snapshot-seq-bucket
-//         |              |
-//         |              |-- <sequence num>
-//         |              |        |
-//         |              |        |-- batch-snapshot-batch: <batch snapshot>
-//         |              |
-//         |              |-- <sequence num>
-//         |              |        |
-//         |             ...      ...
-//         |
-//         |-- batch-snapshot-batchid-index-bucket
-//                       |
-//                       |-- <batch id>: <sequence-num>
-//                       |-- <batch id>: <sequence-num>
-//                       |
-//                      ...
+//
```

Comment:
nit: is this empty line intentional?

Reply:
yes, the comment formatted adds it automatically